### PR TITLE
Improvement and typo in release notes

### PR DESCRIPTION
**Pull request by moy**

### CODE REVIEW

File: RELEASE_NOTES [hunk 1]
```diff
@@ -17,12 +17,12 @@ The Big Ones:
...
+- In Highlight reconstruction module, the mode LCh reconstruction was completely rewritten, and is no longer useless!
 - A new tool, darktable-chart, that goes along with the CLUT module, to create styles for the latter
 - Exposure fusion in the basecurve module to lower the dynamic range of images (https://www.darktable.org/2016/08/compressing-dynamic-range-with-exposure-fusion/)
 - Add new raw overexposure indication (https://www.darktable.org/2016/10/raw-overexposed/)
 - Darkroom: the preview (in top-left corner) is now no longer pre-demosaiced, thus the preview, color picker, histogram show the proper results when there is highlight clipping.
-- Add undo/redo support for the darkroom
+- Add undo/redo support for the darkroom (Ctrl-z to undo, Ctrl-y to redo). This comes in complement to the history stack: undo/redo can walk through several successive changes within the same module (while the history stack only records one entry in this case). Also, undo is able to recover a history stack lost by mistake (e.g. by pressing "compress history" with the bottom of the stack selected).
```
Comment (LebedevRI) [inline RELEASE_NOTES]: "successive" looks strange there, i even had to google it
subsequent?

Comment (moy) [inline RELEASE_NOTES]: Not a native speaker, and I may well have added a french touch, but looking at a dictionary it seems "successive" is the right word here (for "Coming one after the other in a series"), while subsequent applies to one modification coming after another, not the set of modification. Perhaps "consecutive" is more common (synonym for successive)?

Comment (moy) [inline RELEASE_NOTES]: Fixed.

**Comment by houz** [comment]
Such an elaborated description of a feature belongs in the usermanual and not the release notes. Please feel free to add things to the file though.

**Comment by moy** [comment]
> Such an elaborated description of a feature belongs in the usermanual and not the release notes.

I strongly disagree. To me, the release notes' main audience is existing dt users (please correct me if this is not the case), which should already have read the manual (if not, you're not going to convince them by publishing cryptic release notes IMHO). For an example of release notes without cryptic message, see e.g. https://about.gitlab.com/2016/08/22/gitlab-8-11-released/. Is is really that bad for readers?

To them, a sentence like "Add undo/redo support for the darkroom" brings essentially no information. They don't get a hint about where to get further information (are you expecting them to re-read the complete manual to find out?), nor any on how to use the feature. For example, @TurboGit posted an announce on [darktable-fr](https://darktable-fr.tuxfamily.org/forums/sujet/undoredo/) about his feature, and one day later the question "what is it, how do I access it?" was asked. The same question will rather obviously arise if the release notes get posted without any details. With my patch, the reader gets the basic info: what is it, why should I be happy about it, and how do I use it (shortcuts).

Now, let me ask: what is the added value of *not* having these explanations in the release notes.

**Comment by houz** [comment]
I agree that we should have a nice overview of new features with explanations. I liked the one Inkscape had a while ago with animated GIFs showcasing things. But those don't belong in a changelog going with the release.

**Comment by moy** [comment]
> I agree that we should have a nice overview of new features with explanations.

FYI, I wrote such detailed text for previous releases on linuxfr.org: http://linuxfr.org/news/darktable-2-0-traitement-et-gestion-de-photographies and http://linuxfr.org/news/darktable-1-6-traitement-de-photos-developpement-d-images-raw. Understanding the release notes was not easy and this year I thought I'd try to tackle the problem upstream by making the release notes easier to understand, to make my task easier if I get time to write an article in french, and also make it easier for other people to write articles in other languages.

I thought the short and sometimes cryptic release notes were this way because no one took time to make them better, hence I stepped in to try to help. I did not know it was deliberate, and I still disagree with it. If you agree that a nice overview is good, you're not helping to achieve the goal by refusing improvements to release notes.

> But those don't belong in a changelog going with the release.

I never got the concept of "changelog". You use Git, you already have the detailed changelog without having to write a text for this. You use redmine and you already have the list of bugs closed during a timeframe. The internal documentation of "what changed" is already there, you don't need RELEASE_NOTES for that.

The text for which I sent a patch is the one which is posted on the users mailing-list, and in the news section of the main website after the release and pre-releases. Typically places where final users expect to get explanations about new features IMHO.

Actually, you didn't answer my question: what is the downside of having the explanations I propose in the text?

**Comment by LebedevRI** [comment]
@moy i do like such write-ups, but i also mostly agree with @houz, they just don't seem to fit into `RELEASE_NOTES`, "TL;DR"

Also, they may be of some interest for other than the `RELEASE_NOTES`, maybe **usermanual** should have some kind of overview of dt releases, and it should be there?

**Comment by houz** [comment]
The changelog also gets added to packages, for example in `/usr/share/doc/darkabel/changelog.gz` on Debian. That file is intended to be a short and concise list. The fact that we use the same list on the website is because no one cared enough to write more so far. If you want to do that: great! But that's for the final release's announcement post.

**Comment by moy** [comment]
> The changelog also gets added to packages, for example in /usr/share/doc/darktable/changelog.gz on Debian

Well, first:

```
$ cat /etc/debian_version 
8.5
$ zcat /usr/share/doc/darktable/changelog.gz
see webpage: http://www.darktable.org/
```

And honestly, I'm happy with that, I just never use these files

Then, assuming you actually have `RELEASE_NOTES` as /usr/share/doc/darktable/changelog.gz (which probably does happen on some packages), I still have the same question I already asked many times: what is the downside of having understandable explanations there? Maybe it's time to answer that question ...

> If you want to do that: great!

Your proposal is to duplicate the effort by witting twice the same text. Then, thanks for the proposal, but no thanks, I'm not going to do that.

**Pull request closed without merging**